pkg/testingutil/survey_ui.go

```diff
@@ -1,13 +1,16 @@
+// +build !windows
```

Comment:
Skipping build. Refer : https://github.com/openshift/odo/issues/2630 for more details and same issue will be tracked in th e mentioned link.